dist/style.css

```diff
@@ -0,0 +1,171 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Lobster+Two&family=Roboto:wght@1,300&display=swap');
+
+*{
+    font-family:  'Lobster Two', cursive, 'Roboto', sans-serif;
+    padding: 0;
+    margin: 0;
+    /* color: white; */
+    color: black;
+    font-size: 1em;
+}
+
+#content{
+    height: 120vh;
+    width: 100vw;
+    background-color: white;
+    overflow: auto;
+}
+
+h1{
+    font-family: 'Lobster Two', cursive;
+    font-size: 3em;
+    padding-top: 10px;
+}
+
+h1, h3{
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+/* .grid{
+    display: grid;
+    width: 100%;
+   grid-template-columns: 40% 50%;
+} */
+nav{
+    display: flex;
+    align-items: center;
+    position:static;
+    padding: 3px;
+    padding-right: 70px;
+    justify-content: space-between;
+    background-color: black;
+  
+}
+nav li a{
+    text-decoration: none;
+    color: white;
+}
+
+nav img{
+    width: 100px;
+}
+
+.home{
+    color: white;
+}
+ul{
+    display: flex;
+    justify-content: end;
+    list-style-type: none;
+    gap: 2em;
+}
+
+ul a:hover{
+    font-size: 1.2em;
+}
+
+img{
+    padding: 10px;
+    width: 35%;
+}
+
+img:hover{
+    cursor: pointer;
+}
+
+/* .captionDiv{
+    padding-top: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+} */
+
+
+
+footer{
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    height: 10%;
+    background-color: black;
+    
```

Comment:
This empty line. can be removed. Please fix this and other similar places.
You can check Google's js style guide as a guide for yours: https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace